app/Services/Transcoding/WebDAVTranscodingStrategy.php

```
@@ -0,0 +1,52 @@
+            $destination,
+            $bitRate,
+            File::hash($destination),
+            File::size($destination),
+        );
+
+        File::delete($tmpSource);
+
+        return $destination;
+    }
+
+    public function deleteTranscodeFile(string $location, SongStorageType $storageType): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$storageType'. (undefined)

(UnusedFormalParameter)